src/en/overlay/pijuice-hat.md

```diff
@@ -40,15 +42,33 @@ PiJuice is a fully uninterruptable / uninterupted power supply that will always
 
-The board has a Raspberry Pi HAT compatible layout, with onboard EEPROM (you can disable the EEPROM if you want also). It has been designed for the Raspberry Pi A+, B+, 2B, 3B and 3B+ but it is also electrically compatible with the Raspberry Pi Zero v1.3 and Zero W v1.1 or any other Pi.
+The board has a Raspberry Pi HAT compatible layout, with onboard EEPROM (you can disable the EEPROM if you want also). It has been designed for the Raspberry Pi A+, B+, 2B, 3B, 3B+ and 4B, Raspberry Pi Zero v1.3, Raspberry Pi Zero Wireless, and any other Raspberry Pi with 40 GPIO pins.
 
 * Onboard 1820 mAh off the shelf Lipo / LiIon battery for ~4 to 6 hours in constant use (with support for larger Lipo Battery of 5000 or 10,000 mAH+ to last up to 24 hrs +)
 * A Full Uninterruptable Power Supply solution
-* Designed for the Raspberry Pi A+, B+, 2B, 3B and 3B+ but also compatible with Raspberry Pi Zero v1.3 and Raspberry Pi Zero Wireless.
+* Designed for the Raspberry Pi A+, B+, 2B, 3B, 3B+ and 4B but also compatible with Raspberry Pi Zero v1.3 and Raspberry Pi Zero Wireless.
 * Integrated Real Time Clock
 * Onboard intelligent on/off switch
 * Low power deep-sleep state with wake on interrupt/calendar event
 * Programmable multi-colored RGB led (x2) and buttons (x3) with super simple user-configurable options
 * Hardware watchdog timer to keep your Raspberry Pi on and working in mission-critical remote applications
 * Full power management API available to Raspberry Pi OS with auto shutdown capability when running low on batteries
 * Low profile design to fit inside many existing Raspberry Pi cases
+
+##Technical Information
+
+* BP7X battery - original battery from Motorola Droid 2 (A955) - 1820mAh battery
+* Microcontroller is an ST Micro STM32F030CCT6 ARM Cortex-M0, 48MHz, F64KB, R8KB, I2C, SPI, USART, 2.4-3.6V
+* Charge IC - BQ24160RGET Charger IC Lithium-Ion/Polymer, 2.5A, 4.2-10V
+* Fuel gauge IC - LC709203FQH-01TWG Battery Fuel Gauge, 1-Cell Li-ion, 2.8%
+* EEPROM - CAT24C32WI-GT3 EEPROM, I2C, 32KBIT, 400KHZ, 1V7-5V5
+* Optional spring pin - Mil-Max 0929-7-15-20-77-14-11-0
+* Compatible with any 4 pin battery on board that can be used with 00-9155-004-742-006 battery contacts from AVX including the BP7X, BP6X, and any compatible batteries including the 1600mAh and 2300mAh ones from CameronSino (CS-MOA853SL and CS-MOA855XL)
```

Comment:
I'm not sure all the technical details you're adding in the bullet points above are really appropriate for the pinout.xyz webpage? :man_shrugging: 
ping @PiSupply and @shawaj

Reply:
I'm never sure which way to lean on this- but I usually let bare technical details slide because they're useful and meaningful and because, in some sense, Pinout's boards section is intended as a discovery tool as much as a documentation tool.

See, for example: https://pinout.xyz/pinout/expander_pi

Reply:
Sorry, but I don't understand completely both comments: are you saying less detail is better?
I wouldn't be a problem to me to do it if it is the way of pinout.xyz. Please clarify on it...

Reply:
Happy with whatever is standard for pinout. From our side we have no issue with this extra data being there if it is useful but it is certainly not essential to knowing the pinout. So either works 😁

Reply:
After having a little time, changed as @Gadgetoid suggested, adding only some useful information, deleting the rest.